.github/pull_request_template.md

```diff
@@ -13,16 +13,6 @@ Closes: #XXX
 
 *(E.g.: This pull request improves documation of area A by adding ....*
```

Comment:
```suggestion
*(E.g.: This pull request improves documentation of area A by adding ....*
```

Reply:
I'm going to apply this to save everyone time waiting on CI